pmu/README.md

```diff
@@ -0,0 +1,14 @@
+# PMU and adaptive PEBS basic functional check
+
+
+## Usage
+1. Make sure latest perf is there.
+2. Run case as
+```
+./pmu_tests.sh -t <case name>
+
+```
+./apebs_tests.sh -t <case name>
```

Comment:
This line should be put in the code block?

Reply:
> This line should be put in the code block?

Updated.